main.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var join = require('path').join;
+
+module.exports = {
+  extends: join(__dirname, '_base.js'),
```

Comment:
Not sure why, but using "./_base.js" here didn't work for me.
Feels like an ESLint bug. I'll try to triage and submit a bug shortly.